runtime/calamari/src/diff_gas_fees.rs

```diff
@@ -0,0 +1,2169 @@
+            dispatch_info,
+            call_len,
+        ));
+
+        // pause_pallets
+        let call = crate::Call::TransactionPause(pallet_tx_pause::Call::pause_pallets {
+            pallet_names: vec![vec![1u8; 32], vec![2u8; 32]],
+        });
+        let (dispatch_info, call_len) = get_call_details(&call);
+        calamari_runtime_calls.push(("pallet_tx_pause", "pause_pallets", dispatch_info, call_len));
+
+        // unpause_pallets
+        let call = crate::Call::TransactionPause(pallet_tx_pause::Call::unpause_pallets {
+            pallet_names: vec![vec![1u8; 32], vec![2u8; 32]],
+        });
+        let (dispatch_info, call_len) = get_call_details(&call);
+        calamari_runtime_calls.push((
+            "pallet_tx_pause",
+            "unpause_pallets",
+            dispatch_info,
+            call_len,
+        ));
+    }
+
+    // pallet_utility
+    {
+        // batch
+        let call = crate::Call::Utility(pallet_utility::Call::batch { calls: vec![] });
+        let (dispatch_info, call_len) = get_call_details(&call);
+        calamari_runtime_calls.push(("pallet_utility", "batch", dispatch_info, call_len));
+
+        // as_derivative
+        let call = crate::Call::Utility(pallet_utility::Call::as_derivative {
+            index: 1,
+            call: Box::new(call),
+        });
+        let (dispatch_info, call_len) = get_call_details(&call);
+        calamari_runtime_calls.push(("pallet_utility", "as_derivative", dispatch_info, call_len));
+
+        // batch_all
+        let call = crate::Call::Utility(pallet_utility::Call::batch_all { calls: vec![] });
+        let (dispatch_info, call_len) = get_call_details(&call);
+        calamari_runtime_calls.push(("pallet_utility", "batch_all", dispatch_info, call_len));
+
+        // dispatch_as
+        let origin: crate::Origin = frame_system::RawOrigin::Signed(ALICE).into();
+        let as_origin: <crate::Origin as frame_support::traits::OriginTrait>::PalletsOrigin =
+            origin.caller().clone();
+        let call = crate::Call::Utility(pallet_utility::Call::dispatch_as {
+            as_origin: Box::new(as_origin),
+            call: Box::new(call),
+        });
+        let (dispatch_info, call_len) = get_call_details(&call);
+        calamari_runtime_calls.push(("pallet_utility", "dispatch_as", dispatch_info, call_len));
+
+        // force_batch
+        let call = crate::Call::Utility(pallet_utility::Call::force_batch { calls: vec![] });
+        let (dispatch_info, call_len) = get_call_details(&call);
+        calamari_runtime_calls.push(("pallet_utility", "force_batch", dispatch_info, call_len));
+    }
+
+    let mut all_extrinsics_gas_fees = vec![];
+    t.execute_with(|| {
+        for (pallet_name, extrinsic_name, dispatch_info, call_len) in calamari_runtime_calls {
+            let fee = TransactionPayment::compute_fee(call_len, &dispatch_info, 0);
+            let float_gax_fees = Multiplier::try_from(fee).unwrap().div(decimal).to_float();
```

Comment:
This does integer division and creates a float after, truncating at DECIMAL.
Not a big deal i guess but it could raise false positives for small/cheap TXNs
maybe convert to float before dividing

Reply:
also `gax`

Reply:
I will add one more filed that would be more readable, but for fees comparison, I will use that tx fees without removing decimal.